### DISCUSSION TITLE
Not work with NOCAPTCHA at Python 3.6 (OS X) [Solved]

### DISCUSSION THREAD

**Discussion by 1vank1n** [django-recaptcha/django-recaptcha#253] [Q&A]
It's mini howto for those have this problem.

Open a terminal and take a look at:

`/Applications/Python 3.6/Install Certificates.command`

Python 3.6 on MacOS uses an embedded version of OpenSSL, which does not use the system certificate store. More details [here](https://bugs.python.org/issue28150).

**Comment by webbyfox** [comment]
Thanks

**Comment by flavoi** [comment]
I'm not sure, what's the solution?
